netlify-cms.config.yml

```
@@ -95,3 +95,62 @@ collections:
     label: Past
 
   preview_path: api/preview?page=about&ref=cms/team/{{slug}}
+
+
+- name: history
+
+  folder: history
+  extension: md
+  format: yaml-frontmatter
+
+  media_folder: '/images/history'
+
+  create: true
+  delete: true
+
+  label: History
+  label_singular: event
+  summary: '{{date}} - {{title}}'
+
+  fields:
+  - name: date
+    label: Date
+    widget: datetime
+    format: DD.MM.YYYY
+    date_format: 'DD.MM.YYYY'
+    time_format: ''
+    default: ''
+    hint: >
+      All events on the page are sorted in chronologic order.
+      The day is not displayed, therefore it doesn't have to be correct.
+    picker_utc: true
+  - name: title
+    label: Event
+    widget: string
+    hint: >
+      The title for the event
+  - name: body
+    label: Comment
+    widget: markdown
+    buttons: [bold, italic, link, quote, bulleted-list, numbered-list]
+    hint: >
+      Try to keep this concise
```

I am not sure the list options fit there but let's keep them. Better if needed in the future.

```suggestion
  - name: body
    widget: markdown
    buttons: [bold, italic, link, quote, bulleted-list, numbered-list]
    label: Description
    hint: >
      Try to keep this concise, it should be approximately 2-3 lines on the website.
```

About "concise" – very good remark, thank you 👍 Note, you can add a character limit to the input but I am not sure this is needed here.